.github/workflows/rust.yml

```
@@ -60,3 +60,18 @@ jobs:
 
       - name: Check clippy
         run: cargo +nightly clippy --all-targets --all-features --verbose -- -D warnings
+
+  coverage:
+    runs-on: ubuntu-latest
+    container:
+      image: xd009642/tarpaulin:develop-nightly
+      options: --security-opt seccomp=unconfined
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Generate code coverage
+        run: cargo +nightly tarpaulin --verbose --all-features --workspace --timeout 120 --out Xml
```

I'd like to try to keep this option

```suggestion
        run: cargo +nightly tarpaulin --engine llvm --verbose --all-features --workspace --timeout 120 --out Xml
```